srcs/juloo.keyboard2/Keyboard2View.java

```diff
@@ -309,6 +309,9 @@ void drawKeyFrame(Canvas canvas, float x, float y, float keyW, float keyH,
     float w = isKeyDown ? _theme.keyBorderWidthActivated : _theme.keyBorderWidth;
     float w2 = _theme.keyBorderWidth / 2.f;
     _tmpRect.set(x + w2, y + w2, x + keyW - w2, y + keyH - w2);
+    // Set keys opacity
+    _theme.keyBgPaint.setAlpha(_config.keyOpacity);
+    _theme.keyDownBgPaint.setAlpha(_config.keyActivatedOpacity);
```

Comment:
You might want to make the borders transparent also:

```
_theme.keyBorderPaint.setAlpha(_config.keyOpacity);
```

Also, this needs to be done only once and can be moved to the constructor of the class.